packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/utils/udt-name-mapper.util.ts

```diff
@@ -0,0 +1,38 @@
+import { FieldMetadataSettings } from 'src/engine/metadata-modules/field-metadata/interfaces/field-metadata-settings.interface';
+
+import { FieldMetadataType } from 'src/engine/metadata-modules/field-metadata/field-metadata.entity';
+
+export const mapUdtNameToFieldType = (udtName: string): FieldMetadataType => {
+  switch (udtName) {
+    case 'uuid':
+      return FieldMetadataType.UUID;
+    case 'varchar':
+      return FieldMetadataType.TEXT;
+    case 'bool':
+      return FieldMetadataType.BOOLEAN;
+    case 'timestamp':
+    case 'timestamptz':
+      return FieldMetadataType.DATE_TIME;
+    case 'integer':
+    case 'int2':
+    case 'int4':
+      return FieldMetadataType.NUMBER;
+    default:
+      return FieldMetadataType.TEXT;
```

Comment:
I'm wondering if we should throw here instead to avoid side effects with unhandled types

Reply:
Good point. We would throw often, but at least we will know when a type is not supported. Will do in another PR, I should look for all types available